content/en/wiki-tut-node.md

```diff
@@ -10,121 +10,109 @@ sidebar_label: Running a node
 
 #### 1.Compile from source
 
-- Compiler environment configuration. Refer to sections 4.1 to 4.1.3 https://github.com/darwinia-network/darwinia#41-hacking-on-darwinia
-- Start command
-- Enter the darwinia root directory
+- Rust environment preparation. Refer to https://github.com/darwinia-network/darwinia#41-hacking-on-darwinia
+
+- Download source
+
+  https://github.com/darwinia-network/darwinia/archive/v0.7.0.zip
+
+- Enter the darwinia source directory
+
 - cargo build --release
-- The compiled executable file darwinia (.exe) can be found under darwinia / target / release
+
+- The compiled executable file darwinia (.exe) can be found under darwinia target/release
 
 #### 2.Download the compiled executable file
 
-- macOS Catalina: https://github.com/darwinia-network/darwinia/releases/download/v0.x.x/darwinia-catalina.tar.gz
-- ArchLinux: https://github.com/darwinia-network/darwinia/releases/download/v0.x.x/darwinia-archlinux.tar.gz
-- Ubuntu: https://github.com/darwinia-network/darwinia/releases/download/v0.x.x/darwinia-ubuntu.tar.gz
-- CentOS: https://github.com/darwinia-network/darwinia/releases/download/v0.x.x/darwinia-centos7.tar.gz
-- Docker: `docker pull darwinianetwork/darwinia:release-v0.x.x`
+https://github.com/darwinia-network/darwinia/releases/tag/v0.7.0
```

Comment:
Sadly, I test failed to download the  code package from this link which refers to 404 page.

Reply:
It's for the final release